src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.csproj

```diff
@@ -4,6 +4,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\..\Common\src\System\Obsoletions.cs" Link="Obsoletions.cs" />
```

Comment:
```suggestion
```
The ref files are auto-generated and have all strings inlined.

https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md